ee/clickhouse/sql/actions.py

```diff
@@ -1,13 +1,7 @@
 ACTION_QUERY = """
 SELECT
     events.uuid,
-    events.event,
-    events.properties,
-    events.timestamp,
-    events.team_id,
-    events.distinct_id,
-    events.elements_chain,
-    events.created_at
+    events.distinct_id
```

Comment:
Since this is only used in tests (and in one specific file only), worth removing from here and putting it either in tests/ folder, or directly inline in test_action::query_action ?

Reply:
Sure, will do in a follow-up given I have a weird chain PR already in works.